pkg/datapath/iptables/iptables.go

```diff
@@ -377,24 +377,22 @@ func (m *Manager) Start(ctx cell.HookContext) error {
 		m.disableIPEarlyDemux()
 	}
 
-	if err := m.modulesMgr.FindOrLoadModules(
-		"ip_tables", "iptable_nat", "iptable_mangle", "iptable_raw", "iptable_filter",
-	); err != nil {
-		m.logger.WithError(err).Warning(
-			"iptables modules could not be initialized. It probably means that iptables is not available on this system")
+	for _, table := range []string{"nat", "mangle", "raw", "filter"} {
```

Comment:
FindOrLoadModules attempts to load the specified module if it is not already loaded. However, the new implementation does not do this, so there seems to be a functional difference. Could this cause a degrade?

Reply:
> This is done by the iptables executable itself.

Oh, I didn't know it. Thanks for the explanation!